src/jsii-project.ts

```diff
@@ -92,6 +92,14 @@ export interface JsiiJavaTarget {
   readonly javaPackage: string;
   readonly mavenGroupId: string;
   readonly mavenArtifactId: string;
+  /**
+   * Used in maven settings for credential lookup (e.g. use github when publishing to GitHub). Defaults to ossrh for Maven Central.
+   */
```

Comment:
Default should be explicitly marked like so:

```suggestion
  /**
   * Used in maven settings for credential lookup (e.g. use github when publishing to GitHub).
   * @default "ossrh" Defaults to Maven Central.
   */
```